scripts/eject.js

```diff
@@ -75,6 +75,8 @@ prompt(
       .replace(/^\/\*\*(\*(?!\/)|[^*])*\*\//, '')
       // Remove license header from AppleScript
       .replace(/^--.*\n/gm, '')
+      // Remove dead code on eject
+      .replace(/\/\/ @remove-on-eject-begin([\s\S]*?)\/\/ @remove-on-eject-end/g, '')
```

Comment:
> The current RegExp /^\/\*\*(\*(?!\/)|[^*])*\*\// removes all multiline JS comments, which could contains useful information or other License.

This is not correct. It only removes the first multiline comment (because it doesn't have /g at the end). So all the other multiline comments on the file will be preserved just fine.